PeerioTechnologies/peerio-mobile
# chore: condensed fileSystemDescription3 to one string [branch ch8401]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8401/mobile-change-file-migration-description-to-a-single-string-instead-of-separating-the-download-link
#### Testing instructions  
Check that file migration dialog "download" button works. The migration text in check is invoked when use has legacy files to be shared. (I tested this by mocking the conditions for the dialog to show).
Note: If by the time this is being tested the icebear branch doesnt exist, it means it was probably merged to dev.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
